### ISSUE TITLE
Project Abandoned?

### ISSUE THREAD

**Issue by alexwender1** [segler-alex/radiobrowser#141]
Does anyone know why it's all stopped?

I emailed Segler and got no answers.

**Comment by segler-alex** [comment]
it is not abandoned, radiobrowser infrastructure is composed of multiple projects and i am currently working on the backend server part. i accept pull requests if you want to help

**Comment by alexwender1** [comment]
A few weeks ago, I emailed you ... I do not know if you received it.

Reintero that I am ready to help in the correction of radios (name, logo, links). Even because, I do not have advanced knowledge in programming.

I take the space, to leave a tip ... I see many applications accessing your database, without reference. I discovered this after using the Odio App (https://odio.io)... which explained which database I was accessing.

It would be necessary to impose a guideline, so that the applications provide the function to insert new stations to the database ... as well as, insert the function in which the user can notify some incorrect information (that will be evaluated by the RadioBrowser team later).
